src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/notifications/NotificationFactory.java

```diff
@@ -0,0 +1,41 @@
+package com.wikia.webdriver.pageobjectsfactory.pageobject.notifications;
+
+import com.wikia.webdriver.common.core.helpers.User;
+import com.wikia.webdriver.elements.mercury.components.discussions.common.PostEntity;
+
+public class NotificationFactory {
+
+  public static Notification getPostReplyNotification(User user, PostEntity.Data post) {
+    return getPostReplyNotfication(user.getUserName(), post.getTitle());
+  }
+
+  public static Notification getPostReplyConsolidatedNotification(User lastActor, int remainingActors, PostEntity.Data post) {
+    String actor = String.format("%s and %d other users", lastActor.getUserName(), remainingActors);
+    return getPostReplyNotfication(actor, post.getTitle());
+  }
+
+  private static Notification getPostReplyNotfication(String user, String postContent) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused private "getPostReplyNotfication" method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AUnusedPrivateMethod)